getting_started/build-guide/build-from-scratch.rst

```diff
@@ -101,6 +101,17 @@ Clone, build, and install alsa-utils.
    ./gitcompile
    sudo make install
 
+In case of alsa-lib linking errors you can try to re-build it with libdir parameter.
+
+.. code-block:: bash
+
+   cd ../alsa-lib
+   ./gitcompile --prefix=/usr --libdir=/usr/lib/x86_64-linux-gnu/
+   sudo make install
+   cd ../alsa-utils
+   ./gitcompile --prefix=/usr --with-curses=ncurses --disable-xmlto --disable-bat
+   sudo make install
+
```

Comment:
we shouldn't recommend a make install in the first place, it's a terrible idea to override the distro settings. we should use /usr/local here.

Reply:
@mwasko @plbossart fwiw we can easily use custom alsa versions (alongside overiding other common tooling) in our docker container. This shoudl be the preffered way too, as it doesn't break a developers host PC.

Reply:
@plbossart current build from scratch guide recommend to make install and my tips only provide info how to resolve eventual ALSA compile linking errors. I can add some warning message that this operation will override system ALSA library.

There is always an option to use docker as @lgirdwood  wrote.

Reply:
warning message is needed. Lets the users know this will overwrite their ALSA install and will likely break things and should only be done if they know what they are doing etc. Preference is to install ALSA locally or use docker.

Reply:
@lgirdwood Is everyone in agreement with the changes?

Reply:
@deb-intel as recommended I have added commit with warning note related to alsa-lib install operation

Reply:
Thanks @mwasko. I will merge this when it is approved by @plbossart or @lgirdwood.